README.md

```diff
@@ -33,3 +33,27 @@ Include diagrams in the site:
 * Update the relevant markdown file to incorporate diagrams into the site.
 * 
 * `git commit` changes
+
+### Run the application locally
+
+Running this application is fairly manual at the moment and requires some technical expertise.  In summary, you can run the application on your own computer by downloading it to your computer, setting up a few dependencies and executing a number of shell commands to start desired components. You will need on your machine:-
+* [postgres](https://www.postgresql.org/download/) installed on the target machine
+* [Docker](https://docs.docker.com/get-docker/)
+* [Node](https://nodejs.org/en/download/) to a GUI/front end application such as Plan Limits (which is a React application)
+* Java and [Gradle](https://gradle.org/install/)
+* And of course **git** to clone the repo to your machine
+
+With above in place, clone the repo to your machine and do the following in the root of your local repo to run the Plan Limits application:-
+1. `cd packages/Manager`
+2. `./batect runSupportServices`
+3. Create a new shell session in the same folder
+4. `./gradew bootRun`
+5. In a new shell, go to `packages/PlanLimitUI`
+6. npm install
+7. `cp .env.local.template .env.local`
+8. Go to (LINZ Basemaps)[https://basemaps.linz.govt.nz/@-41.8899962,174.0492437,z5) and get a 90 day API key
+9. Edit .env.local, adding your basemaps API key from the last step.  It should be be obvious where to insert it and should be a double-quoted string.
+    eg, `VITE_LINZ_API_KEY="c01hnehzqpjbep1x6kgf8ey8wxw"`
+10. Ensure there are no instances of postgres running on your system
```

Comment:
This point is probably not needed. Because why call out Postgres and not the other things that might cause port conflicts? ... but also it should be step 0 because step 2 is the one that starts a Postgres.

Reply:
Yeah, true.  It was a particular problem I encountered, so I included it.  I could imagine a lot of devs having postgres running on the default port.  Maybe we could change the port that the batect database container exposes?

Reply:
What would you think of calling out all the ports that will be used and leaving it to the dev to suss out if they have other things running?

Reply:
Yeah, that sounds reasonable.  I have added to the main readme.  I've just mentioned 8080, 5432 (postgres), and 9092 (Kafka) for now.  Any others I should add?